app/helpers/application_helper.rb

```diff
@@ -3,4 +3,25 @@ module ApplicationHelper
   def flash_class(level)
     level == :notice ? "info" : level.to_s
   end
+
+  def markdown(text)
+    options = {
+      filter_html:     true,
+      hard_wrap:       true,
+      link_attributes: { rel: 'nofollow', target: "_blank" },
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.